### PR TITLE
Dark Skies Part 4: Overrides

### PR DESCRIPTION
**Pull request by ephemeralstoryteller**
#### Summary
```SUMMARY: Mods "Part 4 of Dark Skies Above, adds override files"```

#### Purpose of change
Adds most of the override files, except for evac_shelter and scenarios, which were fairly big.

#### Describe the solution
 - For brevity: Most files are edits to descriptions, stripping out references to zombies or other base game lore.
 - Cemetery: edits out the bone seer until I get a chance to work with her more explicitly
 - Carnivore also tweaks the names of tainted meat and mutant meat to represent alien flesh or heavily mutated flesh, and flesh from creatures tampered with by the aliens.
 - Newspaper was changed to call Dark Skies's own list of snippets.
 - Mapgen: Lowers the frequency of toxic waste dumps, because they were spawning quite frequently with some of the other locations being blacklisted.
 - Monsters gives manhacks the "alien" category, so they will spawn if called by enemies that deploy manhacks. It also nixes the "wildlife" category for giant rattlesnakes and naked mole rats.

#### Testing
Tested copy-from to cut down on the diffs of professions.json, it worked.

### CODE REVIEW

**Comment by ephemeralstoryteller** [comment]
After running this through the web linter several times, I've come to the conclusion that I don't know how to fix the JSON formatting. If anyone else has an idea of what the problem, do highlight it for me.

File: data/mods/Dark-Skies-Above/overrides/items/electronic.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "delete": { "flags": [ "USE_UPS" ] }
+  },
+  {
+    "id": "smart_phone_flashlight",
+    "copy-from": "smart_phone_flashlight",
+    "type": "TOOL",
+    "name": { "str": "smartphone - Flashlight", "str_pl": "smartphones - Flashlight" },
+    "delete": { "flags": [ "USE_UPS" ] }
+  }
+]
```
Comment (AMurkin) [inline data/mods/Dark-Skies-Above/overrides/items/electronic.json]: Line-endings difference.
I just selected all lines and clicked `Insert a suggestion` button.
```suggestion
[
  {
    "id": "smart_phone",
    "type": "TOOL",
    "name": { "str": "smartphone" },
    "copy-from": "smart_phone",
    "description": "A popular, fancy smartphone.  Capable of making photos due to integrated camera and illuminating an area as per its flashlight app, assuming it has enough charge.  The smartphone also has a clock app that includes an alarm.  Runs on a small, proprietary power cell that needs a specialized charger. That is to say, is quite difficult to recharge post-Cataclysm.",
    "delete": { "flags": [ "USE_UPS" ] }
  },
  {
    "id": "smartphone_music",
    "copy-from": "smartphone_music",
    "type": "TOOL",
    "name": { "str": "smartphone - music", "str_pl": "smartphones - music" },
    "delete": { "flags": [ "USE_UPS" ] }
  },
  {
    "id": "smart_phone_flashlight",
    "copy-from": "smart_phone_flashlight",
    "type": "TOOL",
    "name": { "str": "smartphone - Flashlight", "str_pl": "smartphones - Flashlight" },
    "delete": { "flags": [ "USE_UPS" ] }
  }
]
```

**Comment by Maleclypse** [comment]
> After running this through the web linter several times, I've come to the conclusion that I don't know how to fix the JSON formatting. If anyone else has an idea of what the problem, do highlight it for me.

In cases where I'm having difficulties I use this link https://jsonlint.com/?json= then once that link tells me I have a valid json I run it through the cata linter.

**Comment by ephemeralstoryteller** [comment]
Thanks for the suggestions! I'm a bit baffled, the json checker reads both what I had and the review as valid, but there's still an issue that I must not be picking up on.

**Pull request merged**